rhapsody/peers/peers-rhapsody5

```diff
@@ -0,0 +1,10 @@
+725ed0c0021e6b8a7d52c862e7c94323dbdfd60a@185.209.229.44:26656,
+704dccd6d3b374f9ef699c026133f6174dc705e8@154.53.55.106:26651,
+86b0be64be9637f6e23ecf475edd164fa07b5665@65.21.134.202:26676,
+c5cbd164de9c20a13e54e949b63bcae4052a948c@138.201.139.175:20956,
+9428068507466b542cbf378d59b77746c1d19a34@157.90.35.151:26657,
+4e7a6d8a3c8eeaad4be4898d8ec3af1cef92e28d@93.186.200.248:26656,
+eaeb462547cf76c3588e458120097b51db732b14@194.163.155.84:26656,
+51af5b6b4b0f5b2b53df98ec1b029743973f08aa@75.119.145.20:26656,
+9a9ed14d71a88354b0383419432ecce70e8cd2b3@161.97.152.215:26656,
+43bca26cb1b2e7474a8ffa560f210494023d5de4@135.181.140.225:26657
```

Comment:
26657 will be an RPC port, not a P2P port, so won't be a valid peer address - I think the 26657's in this list should be 26656 (I think they are also both Quicksilver nodes!)

Reply:
I think so too, but the guys from the chat sent me. Maybe they did port forwarding, but it's wrong. I changed.